node-test/src/test/scala/org/bitcoins/node/NeutrinoUnsupportedPeerTest.scala

```diff
@@ -16,13 +17,21 @@ class NeutrinoUnsupportedPeerTest extends NodeTestWithCachedBitcoindV19 {
   override type FixtureParam = NeutrinoNodeConnectedWithBitcoind
 
   override def withFixture(test: OneArgAsyncTest): FutureOutcome = {
-    val outcomeF: Future[Outcome] = for {
-      bitcoind <- cachedBitcoindWithFundsF
-      outcome = withNeutrinoNodeUnstarted(test, bitcoind)(system,
-                                                          getFreshConfig)
-      f <- outcome.toFuture
-    } yield f
-    new FutureOutcome(outcomeF)
+    if (TorUtil.torEnabled) {
+      // We must skip this test for tor enabled
+      // because bitcoind only supported tor v2 at the time
+      // which is now deprecated and no longer supported by
+      // the tor network
```

Comment:
my suggestion here so we don't have this trivially succeed in the future if we change versions 

```
      // We must skip this test for tor enabled
      // because bitcoind only supported tor v2 at the time
      // which is now deprecated and no longer supported by
      // the tor network

      val f: Future[Outcome] = for {
        bitcoind <- cachedBitcoindWithFundsF
        _ = require(
          bitcoind.version == BitcoindVersion.V19,
          s"Expected bitcoind version 19, got version=${bitcoind.version}." +
            s"If this version is >= 21, we need to remove this if clause as bitcoind 21+ supports tor v2"
        )
        trivialSucceed <- FutureOutcome.succeeded.toFuture
      } yield trivialSucceed

      new FutureOutcome(f)
```

Although now i think about it, all these new versions will have neutrino supported, so feel free to omit